### ISSUE TITLE
Final Peer Review (wmx2)

### ISSUE THREAD

**Issue by william-xiao2** [vp237/IPO_Prediction#7]
This project aims to predict whether an IPO does well using only information present only on the day of the IPO going public, such as sector, number of employees, date, company CEO, etc. The group used techniques such as missing value imputation, logistic regression, and random forests to make The information gleaned in this project can be used to further predict the price of IPOs in the future, should one decide to invest in one in attempts to gain a profit. 

Some things I liked about this project are:
- I liked how the group mentioned that their dataset had 3500+ examples, and that they were using cross-validation and 70/30 train/test splits to prevent overfitting / underfitting. However, I would have liked to see more discussion about how, in the grand scheme of things, 3500 examples is not many. I would have liked to see this acknowledged more in the data analysis, or at least a consideration of bootstrapping.
- The report was organized well and had a good logical flow. It made sense, how it first started with an introduction of the dataset, then a discussion of how missing values were imputed / data was cleaned, then the models used, then reflections on the results, and then conclusions. 
- I liked the analysis used in the balanced accuracy section, and how the group took the time to thoroughly analyze their data and recognize these kinds of adjustments that had to be made in order to not skew their metrics to artificially inflate the performance of their models. 

Some things that I think could be improved are:
- The report seems to be a little hand-wavy about their sources sometimes. For example, they say "According to an expert we consulted...". Who is the expert? What are their qualifications that makes then an "expert"? How do you know it is not a random person on the internet? The report also mentions things like "According to a research paper..." without mentioning its actual title inline. Having it inline would make it much more credible so readers could easily see the paper and assess its credibility.
- The diagrams on the report were sometimes a little hard to read and understand (in particular, all the graphs in section 8). What do each of these graphs correspond to? You can kind of tell from context, but for someone who has never really looked into IPOs before this report, they can be quite confusing. Color coding would help. Labeling each bar withe value for the bar charts might help too. Or honestly, even just using something other than the basic matplotlib plotting functionality. 
- The report picks 10 stocks and graphs their log returns, etc. However, I would have liked to see some textual elaboration on these stocks.  What were the 10 companies? Can you pick one of them and discuss the results in more detail, e.g. why you saw the trends in that company's data / whether the model did well on that company or not, and why?
